sdk/core/Azure.Core.Experimental/src/GeoJson/GeoPoint.cs

```diff
@@ -42,15 +42,18 @@ public GeoPoint(GeoPosition position): this(position, null, DefaultProperties)
         /// </summary>
         /// <param name="position">The position of the point.</param>
         /// <param name="boundingBox">The <see cref="GeoBoundingBox"/> to use.</param>
-        /// <param name="additionalProperties">The set of additional properties associated with the <see cref="GeoObject"/>.</param>
-        public GeoPoint(GeoPosition position, GeoBoundingBox? boundingBox, IReadOnlyDictionary<string, object?> additionalProperties): base(boundingBox, additionalProperties)
+        /// <param name="customProperties">The set of additional properties associated with the <see cref="GeoObject"/>.</param>
+        public GeoPoint(GeoPosition position, GeoBoundingBox? boundingBox, IReadOnlyDictionary<string, object?> customProperties): base(boundingBox, customProperties)
         {
-            Position = position;
+            Coordinates = position;
         }
 
         /// <summary>
         /// Gets position of the point.
         /// </summary>
-        public GeoPosition Position { get; }
+        public GeoPosition Coordinates { get; }
+
+        /// <inheritdoc />
```

Comment:
Shouldn't Chris's change inline the actual comment here (and perhaps elsewhere earlier that I missed till now)?

Reply:
Chris's change inlines `inheritdoc` comments in the resulting XML doc file.